wkhtmltopdf/utils.py

```diff
@@ -46,6 +46,7 @@
                        '--enable-toc-back-links', '--footer-line', '--no-footer-line',
                        '--header-line', '--no-header-line', '--disable-dotted-lines',
                        '--disable-toc-links', '--verbose']
+MULTI_VALUE_OPTIONS = ['--custom-header', '--cookie']
```

Comment:
I think there are a few more tuple-options:

```
$ curl -s https://wkhtmltopdf.org/usage/wkhtmltopdf.txt | grep '<name> <value>'
      --cookie <name> <value>         Set an additional cookie (repeatable),
      --custom-header <name> <value>  Set an additional HTTP header (repeatable)
      --post <name> <value>           Add an additional post field (repeatable)
      --replace <name> <value>        Replace [name] with value in header and
```

Reply:
Thank you @erikvanzijst for pointing these out!